lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -114,6 +116,7 @@ sub list {
     #         0;
     #     }
     # }
+    my @scheduled = $scheduled->search({blocked_by_id => undef})->all;
```

Comment:
I guess eventually we'd need to re-add the possibility of showing which jobs are blocked.

Reply:
Yes. I still want the /tests page to be a little more useful - e.g. by showing all tests and autoupdate on state changes. But that's for later.